src/components/utils/firebase.ts

```diff
@@ -0,0 +1,23 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getAnalytics } from "firebase/analytics";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: "AIzaSyDkWfubVqvH4AaVs4Q3OE1tJ4hpyBCCZVU",
```

Comment:
BRUHHHH, API key commit?? bruh?! get yourself a fucking .env file

Reply:
updating it later